### DISCUSSION TITLE
Did someone deploy the express api shown in the turborepo template example: https://vercel.com/templates/remix/turborepo-kitchensink?

### DISCUSSION THREAD

**Discussion by hseghetti** [vercel/community#6670] [Help]
### Summary

Hi all, I have been trying to deploy the express API shown in this template: https://vercel.com/templates/remix/turborepo-kitchensink

Deploying the frontend apps is straightforward but express server was impossible. 
I have tried creating one project for the express server but following some project settings, the build fails because there is no public folder.

###  I have followed the monorepo deployment steps:

![image](https://github.com/vercel/community/assets/1796268/7653caad-f141-40bc-8f68-e392f262a70e)

**Repository (Fork from the template):** [https://github.com/hseghetti/turborepo-kitchensink](https://github.com/hseghetti/turborepo-kitchensink )

Do you have any experience with something similar? 
Thanks!

### Build Error

![image](https://github.com/vercel/community/assets/1796268/e25e2e73-69ee-446a-be96-7d1f734f0642)

### Steps to Reproduce

1. Clone https://vercel.com/templates/remix/turborepo-kitchensink
2. Try to deploy the template but the express server. 
3. You will have a build error

**Comment by versecafe** [comment]
@hseghetti you shouldn't need to cd around in the build step but you do need to make sure you have a `vercel.json` to actually deploy express to Vercel as it's not a default supported framework
```json
{ "version": 2, "rewrites": [{ "source": "/(.*)", "destination": "/api" }] }
```
should work

**Reply by hseghetti** [reply]
@versecafe thank you for your answer. 
I have removed all the Build & Development Settings and added the vercel.json but I still have the error:

vercel.json -> [https://github.com/hseghetti/turborepo-kitchensink/blob/main/apps/api/vercel.json](https://github.com/hseghetti/turborepo-kitchensink/blob/main/apps/api/vercel.json)

**Error**

<img width="1178" alt="image" src="https://github.com/vercel/community/assets/1796268/8f91ef75-0f8b-4e25-bb58-fdf2a7d372c1">
